core/src/main/java/org/apache/carbondata/core/datastore/page/ColumnPage.java

```diff
@@ -436,6 +439,11 @@ public void putData(int rowId, Object value) {
   public abstract void putShortInt(int rowId, int value);
 
   /**
+   * Set boolean value at rowId
+   */
+  public abstract void putBoolean(int rowId, boolean value);
```

Comment:
Since internally boolean is stored as byte, so you can implement this function to call `putByte` instead of abstract function

Reply:
ok, I have changed it